Plugins/Public/autobuy/Main.cpp

```diff
@@ -288,21 +315,38 @@ void AutobuyInfo(uint iClientID)
 	PrintUserCmdText(iClientID, L"Error: Invalid parameters");
 	PrintUserCmdText(iClientID, L"Usage: /autobuy <param> [<on/off>]");
 	PrintUserCmdText(iClientID, L"<Param>:");
-	PrintUserCmdText(iClientID, L"   info - display current autobuy-settings");
-	PrintUserCmdText(iClientID, L"   missiles - enable/disable autobuy for missiles");
-	PrintUserCmdText(iClientID, L"   torps - enable/disable autobuy for torpedos");
-	PrintUserCmdText(iClientID, L"   mines - enable/disable autobuy for mines");
-	PrintUserCmdText(iClientID, L"   cd - enable/disable autobuy for cruise disruptors");
-	PrintUserCmdText(iClientID, L"   cm - enable/disable autobuy for countermeasures");
-	PrintUserCmdText(iClientID, L"   reload - enable/disable autobuy for nanobots/shield batteries");
-	PrintUserCmdText(iClientID, L"   repair - enable/disable auto-repair");
-	PrintUserCmdText(iClientID, L"   all: enable/disable autobuy for all of the above");
+	PrintUserCmdText(iClientID, L"info - display current autobuy-settings");
```

Comment:
put the indentation back please

Reply:
no no no, because of indentation, console msgs were not appearing to user!

Reply:
why?

Reply:
I think it is inbuilt to game feature
and if I am right... it is because "freelancer creators" made "Console commands" affecting the game starting from space
this is why they make them disappear. No need to show them to the player.

Reply:
fixed the issue by using solution from other plugin. just placed | at the phrase beginning